app/controllers/ems_container_controller.rb

```diff
@@ -17,4 +17,8 @@ def self.table_name
   def index
     redirect_to :action => 'show_list'
   end
+
+  def documantation_url
```

Comment:
@alongoldboim I assume you used this to test it (and produce the screenshot) but until we won't have a link we may even drop it from the patch.